purchase_lead_hour/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Purchase Lead Hour",
+    "summary": "Allows to set more accurate vendor lead delays",
+    "version": "14.0.1.0.0",
+    "category": "Purchases",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "author": "Camptocamp SA, Odoo Community Association (OCA)",
+    "maintainers": ["mmequignon"],
+    "license": "AGPL-3",
+    "installable": True,
+    "auto_install": False,
+    "depends": ["product", "stock"],
```

Comment:
As your module is named `purchase_*` we expect it to depend on `purchase`, or if the vendor delay can be used outside of a purchase flow, another module name could be found maybe.

Reply:
If you're to use `product.supplierinfo` you have to depend on `purchase_stock` as `_get_lead_days` would filter on pull and pull_push rules if not installed.